application/classes/Ushahidi/Console/ProcessExportJobs.php

```diff
@@ -0,0 +1,131 @@
+<?php defined('SYSPATH') or die('No direct script access');
```

Comment:
Could you create this under src/Console and set up the DI in src/Console/Init.php too?
If you do it here its a pain for lumen conflicts